lib/htmltoword/document.rb

```diff
@@ -113,16 +113,14 @@ def transform_and_replace(source, stylesheet_path, file, remove_ns = false)
     def local_images(source)
       source.css('img').each_with_index do |image,i|
         filename = image['data-filename'] ? image['data-filename'] : image['src'].split("/").last
-        ext = ext_from_filename(filename)
+        ext = File.extname(filename).delete(".").downcase
 
         @image_files << { filename: "image#{i+1}.#{ext}", url: image['src'], ext: ext }
       end
     end
 
-    #get extension from filename and clean to match content_types
-    def ext_from_filename(filename)
-      ext = File.extname(filename).delete(".").downcase
-
+    #get content type from extension
+    def content_type_from_extension(ext)
       case ext
```

Comment:
I've been thinking about this and agree with changing the method to `content_type_from_extension`. However I think that we could simplify the method right down to just:

`ext == "jpeg" ? "jpg" : ext`